cmd/kubelet/app/server.go

```diff
@@ -412,13 +431,249 @@ func run(s *options.KubeletServer, kcfg *KubeletConfig) (err error) {
 	return nil
 }
 
+// bootstrapClientCert will request a client cert for kubelet.
+// If the file specified by --kubeconfig does not exist, the bootstrap kubeconfig is used
+// to request a client certificate from the API server.
+// On success, a kubeconfig file referencing the generated key and obtained certificate is
+// written to the path specified by --kubeconfig.
+// The certificate and key file will be stored in /var/run/kubernetes/.
+func bootstrapClientCert(s *options.KubeletServer) error {
+	// Check the if --kubeconfig already has sufficient TLS client info.
```

Comment:
I don't think we need to check all this, just stat the kubeconfig file. if it exists, or you get an error other than `os.IsNotExist(err)`, return. Once the kubeconfig exists, I think the bootstrapper's job is done